### DISCUSSION TITLE
MAPL 2.7.3

### DISCUSSION THREAD

**Discussion by mathomp4** [GEOS-ESM/MAPL#905] [Announcements]
This is a bugfix update to MAPL. Per Issue #899:

> Both MAPL and GOCART assume that grid coordinates are in radians. However, this may not be the case if the grid is provided externally, as it happens for the UFS aerosol component.

This update ensures coordinates are in radians.

---

### Fixed

- Ensure grid coordinates are always provided in radians

<hr /><em>This discussion was created from the release <a href='https://github.com/GEOS-ESM/MAPL/releases/tag/v2.7.3'>MAPL 2.7.3</a>.</em>

**Comment by mathomp4** [comment]
@rmontuoro MAPL 2.7.3 issued with your bugfix.

**Reply by rmontuoro** [reply]
Thanks a lot, @mathomp4!
